docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -700,10 +700,42 @@ You can see them also by typing ```%%cython?`` in IPython or a Jupyter notebook.
 ============================================  =======================================================================================================================================
 
 
+.. _cython-cache:
+
+Cython cache
+============
+
+Cython cache is used to store cythonized ``.c``/``.cpp`` files to avoid running cython compiler on the files which were cythonized before.
+
+.. note::
+
+   Only ``.c``/``.cpp`` files are cached. The C compiler is run every time. To avoid executing C compiler ccache needs to be used.
```

Comment:
```suggestion
   Only ``.c``/``.cpp`` files are cached. The C compiler is run every time. To avoid executing C compiler a tool like ccache needs to be used.
```

(I'm sure ccache isn't the only option here)